test/powershell/Language/Operators/RangeOperator.Tests.ps1

```diff
@@ -0,0 +1,135 @@
+Describe "Range Operator" -Tags CI {
+    Context "Range integer operations" {
+        It "Range operator generates arrays of integers" {
+            $Range = 5..8
+            $Range.count | Should Be 4
+            $Range[0] | Should BeOfType [int]
+            $Range[1] | Should BeOfType [int]
+            $Range[2] | Should BeOfType [int]
+            $Range[3] | Should BeOfType [int]
+        }
+
+        It "Range operator accepts negative integer values" {
+            {
+                -8..-5
+            } |Should Not Throw
+
+            $Range = -8..-5            
+            $Range.count | Should Be 4
+            $Range[0] |Should Be -8

```

Comment:
Formatting - please add space after |

Please add type check as in previous test.